IDE/VS-ARM/README.md

```diff
@@ -0,0 +1,11 @@
+VisualStudio for ARM build
+==========================
+
+VS2017 now offers solution options for Android and iOS applications. It includes
+the GCC cross-compiler suite and property sheets for configuring a project
+for those platforms and libraries for cross-platform mobile application
+development.
+
+The provided solution file will build wolfSSL for linking into an ARM based
+application, _libwolfssl.a_. The _user_settings.h_ file enables some of the
```

Comment:
For Markdown please use asterisk instead of underscore. Should be `libwolfssl.a` and `user_settings.h`. Although _ in GitHub shows with italic.